lib/rest-middleware.js

```diff
@@ -0,0 +1,90 @@
+function restMiddleware (connect, helmet, logger, restifyLinks) {
+    this.config = {};
```

Comment:
I would eliminate this by moving the below 2 functions into the function on line 55.